airflow/config_templates/config.yml

```diff
@@ -1831,7 +1831,7 @@ webserver:
       version_added: 2.3.0
       type: string
       example: ~
-      default: "gantt,landing_times,tries,duration,calendar,graph,grid,tree,tree_data"
+      default: "gantt,landing_times,tries,duration,calendar,graph,grid,tree"
```

Comment:
`tree_data` doesn't exist . Should we keep these legacy ones or redo this exclude list entirely?

Reply:
I am in favour of redoing the list to get rid of legacy stuff

Reply:
+1

Reply:
Ok! I removed it entirely